Code/Controllers/ATLMApplicationController.m

```
@@ -74,7 +75,26 @@ - (void)layerClient:(LYRClient *)client didReceiveAuthenticationChallengeWithNon
     NSLog(@"Layer Client did recieve authentication challenge with nonce: %@", nonce);
     ATLMUser *user = self.APIManager.authenticatedSession.user;
     if (!user) return;
```

Is this just a sanity check? Or is there a legit code path that could lead to this (in which case maybe we shouldn't silently fail)?